app/controllers/client_opportunity_matches_controller.rb

```diff
@@ -20,14 +20,10 @@ def index
     end
 
     # client filter
-    if params[:client_id].present? && (@client = Client.find(params[:client_id].to_i))
-      @matches = @matches.where(client_id: @client.id)
-    end
+    @matches = @matches.where(client_id: @client.id) if params[:client_id].present? && (@client = Client.find(params[:client_id].to_i))
```

Comment:
Side-effect assignments make me itchy. As long as we are changing this, maybe?
```suggestion
    @client = Client.find( params[:client_id].to_i) if  params[:client_id].present?
    @matches = @matches.where(client_id: @client.id) if @client.present?
```

Reply:
yikes, yeah, good catch.  I think that was a bungled rubocop cleanup.